index.d/che-stacks.yaml

```diff
@@ -130,15 +130,15 @@ Projects:
     notify-email: yivantso@redhat.com
     build-context: ./
     depends-on: che-stacks/centos-stack-base:latest
-
-  - id: 13
-    app-id: che-stacks
-    job-id: centos-quarkus-io-jdk8
-    git-url: https://github.com/eclipse/che-dockerfiles
-    git-branch: master
-    git-path: recipes/centos_quarkus.io_jdk8
-    target-file: Dockerfile
-    desired-tag: latest
-    notify-email: mike@vorburger.ch
-    build-context: ./
-    depends-on: che-stacks/centos-jdk8:latest
+#
```

Comment:
Please add a comment here as TODO about why we commented these lines and next actions to be taken.

Reply:
done